nixos/lib/make-disk-image.nix

```diff
@@ -180,9 +180,17 @@ let format' = format; in let
     export NIX_STATE_DIR=$TMPDIR/state
     nix-store --load-db < ${closureInfo}/registration
 
-    echo "running nixos-install..."
-    nixos-install --root $root --no-bootloader --no-root-passwd \
-      --system ${config.system.build.toplevel} --channel ${channelSources} --substituters ""
+    mkdir -m 0755 -p "$root/etc"
+    touch "$root/etc/NIXOS"
```

Comment:
Why is `NIXOS` in allcaps?

Reply:
That file indicates that is a NixOS system (see for example https://nixos.org/nixos/manual/index.html#sec-installing-from-other-distro).

Reply:
We might want to drop that file in the future, and rely on `/etc/os-release`: http://0pointer.de/blog/projects/os-release cc @arianvp @aanderse